src/components/BasicHead.vue

```diff
@@ -65,7 +65,7 @@
     vertical style="margin:0px 16px;"></v-divider>
 
   <v-combobox
-    v-if="tool.search.active && show('search')"
+    v-if="tool.search.active && show('search') && filterIcon"
```

Comment:
use a single combobox and do this instead:

```
:append-icon="filterIcon?mdi-filter:undefined"
```

Reply:
Done